pkg/server/definitionHandler.go

```diff
@@ -117,25 +117,20 @@ func prepareParameterCue(fileDir, fileName string, targetSchemaDir string) error
 	}
 
 	scanner := bufio.NewScanner(f)
-	var startCopyFlag, endCopyFlag bool
+	var withParameterFlag bool
 	for scanner.Scan() {
 		text := scanner.Text()
-		if strings.HasPrefix(text, "parameter: {") {
-			startCopyFlag = true
+		if strings.ReplaceAll(text, " ", "") == "parameter:{" {
```

Comment:
i think you should try regex pattern for this check.

For the following case, it should work but will fail in your way:

```
parameter: abc: string
```

```
parameter: { abc:string, xyz:int}
```

Reply:
Good catch, thanks.